lib/utils/public-key-util.js

```diff
@@ -49,7 +49,7 @@ module.exports = (function () {
 		}, function (error, response, body) {
 			
 			if (error || response.statusCode !== 200) {
-				if (!publicKeysJson) {
+				if (typeof publicKeysJson[publicKeysUrl] === 'undefined' || !publicKeysJson[publicKeysUrl]) {
```

Comment:
Do you need the typeof statement? Look a new suggestion feature 😮 
```suggestion
				if (!publicKeysJson[publicKeysUrl]) {
```